tests/au_login_test.py

```diff
@@ -0,0 +1,22 @@
+import logging
+import os
+
+from hyundai_kia_connect_api.VehicleManager import VehicleManager
+
+
+def test_AU_login():
+    username = os.environ["KIA_AU_USERNAME"]
```

Comment:
Are you interesting in adding your login details as a secret for the repo? This will be needed so GitHub tests can run.  All good either way! 

If interested I'll grant you access to the repo to add the secrets.  Each PR after this will run a test to confirm login still functions for regions we have this setup for.

Reply:
Hm... probably not now but I'll consider it later. I'm still testing a lot of stuff locally and I've locked myself out of my account multiple times now due to spamming the login endpoint and hitting the login timeout 😅 so that wouldn't make for a very good test right now. 

Happy to once this is all more stable though!

Reply:
Will comment this test out for now similar to the CA test.